README.md

```diff
@@ -43,6 +43,11 @@ The javascript API is exposed on a NHSCookieConsent global variable.
 console.log(NHSCookieConsent.VERSION)
 ```
 
+## NoBanner environment variable
```

Comment:
Stick this under the compiling section.

Also: does sense any make in english?

Reply:
me no english well :(